src/open_inwoner/cms/cases/tests/test_plugin_cases.py

```diff
@@ -0,0 +1,86 @@
+from unittest.mock import patch
+
+from django.test import TestCase, override_settings
+
+import lxml
+import requests_mock
+from pyquery import PyQuery as pq
+
+from open_inwoner.accounts.choices import LoginTypeChoices
+from open_inwoner.accounts.tests.factories import UserFactory
+from open_inwoner.openzaak.tests.mixins import ZakenTestMixin
+from open_inwoner.utils.test import ClearCachesMixin
+
+from ...tests import cms_tools
+from ..cms_apps import CasesApphook
+from ..cms_plugins import CasesPlugin
+
+
+@requests_mock.Mocker()
+@patch.object(CasesPlugin, "limit", 2)
+@override_settings(ROOT_URLCONF="open_inwoner.cms.tests.urls")
+class CasesPluginTest(ZakenTestMixin, ClearCachesMixin, TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+        cms_tools.create_apphook_page(CasesApphook)
+
+    def test_cms_plugin_cases_not_rendered_for_anonymous_user(self, m):
+        self.setUpMocks(m)
+
+        # anonymous user
+        html, context = cms_tools.render_plugin(CasesPlugin)
+
+        self.assertIsNone(context["cases"])
+
+        # check that html empty
+        with self.assertRaises(lxml.etree.ParserError) as ctx:
+            pq(html)
+
+        self.assertEqual(str(ctx.exception), "Document is empty")
```

Comment:
@pi-sigma We don't really need lxml and pyquery to check if a string is empty :smile: